client/galaxy/scripts/mvc/history/history-view.js

```diff
@@ -550,14 +539,12 @@ var HistoryView = _super.extend(
 
 //------------------------------------------------------------------------------ TEMPLATES
 HistoryView.prototype.templates = (() => {
-    var mainTemplate = BASE_MVC.wrapTemplate([
-        // temp container
-        "<div>",
-        '<div class="controls"></div>',
-        '<ul class="list-items"></ul>',
-        '<div class="empty-message infomessagesmall"></div>',
-        "</div>"
-    ]);
+    var mainTemplate = () =>
```

Comment:
I think mainTemplate should just be the string literal here (no arrow function), unless I'm missing something in the refactoring.

Reply:
Fair point - I'll give that a try.

Reply:
Actually (and I haven't tested it yet) it may have to be a function even though it takes no params, now that I think about it a little more.  Sorry for the noise, if so!

Reply:
Looks like it is needed. 

<img width="461" alt="screen shot 2017-11-14 at 11 25 24 am" src="https://user-images.githubusercontent.com/216771/32791238-8d81abd2-c92e-11e7-98fc-141e5fdd3fbe.png">

Reply:
Yep, figured.  Good to know, thanks for checking!